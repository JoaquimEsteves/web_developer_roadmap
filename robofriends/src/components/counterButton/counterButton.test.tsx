import React from 'react';


import { shallow, mount, render } from 'enzyme';
import CounterButton from './counterButton';
import '../../setupTests';

it('try out the snapshot yo', () => {
    const mockColor = 'red';
    expect(mount(<CounterButton color={mockColor}/>).debug()).toMatchSnapshot();
})