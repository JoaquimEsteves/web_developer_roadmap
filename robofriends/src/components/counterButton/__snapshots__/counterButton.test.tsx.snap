// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`try out the snapshot yo 1`] = `
"<CounterButton color=\\"red\\">
  <button color=\\"red\\" onClick={[Function]}>
    Count: 
    0
      Yo
  </button>
</CounterButton>"
`;
